indico/modules/attachments/templates/_display.html

```diff
@@ -10,8 +10,8 @@
                     {{ template_hook('event-display-after-attachment', attachment=attachment, top_level=true, has_label=true) }}
                 {%- endfor -%}
             {% else %}
-                <a class="i-button arrow" data-toggle="dropdown"></a>
-                <ul class="dropdown">
+                <a class="i-button arrow" data-toggle="i-dropdown"></a>
```

Comment:
It's not necessary to change `data-toggle`. This is not a CSS class, just an arbitrary `data-` attribute.